ui/job-view/details/RetriggerMultiplePrompt.jsx

```diff
@@ -0,0 +1,108 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import {
+  Button,
+  Input,
+  Modal,
+  ModalBody,
+  ModalFooter,
+  ModalHeader,
+} from 'reactstrap';
+import $ from 'jquery';
+
+import JobModel from '../../models/job';
+import { notify } from '../redux/stores/notifications';
+
+export class RetriggerMultipleClass extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      retriggerTimesValue: 1,
+      retriggerTimesValid: true,
+    };
+  }
+
+  retriggerTimes = () => {
+    const { repoName } = this.props;
+    this.props.toggle();
```

Comment:
Please get the ``toggle`` function out of ``this.props`` with destructuring.  See: https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/destructuring-assignment.md